### PR TITLE
*: Fix address normalization

### PR DESCRIPTION
**Pull request by kakkoyun**
With yet to be release version of Parca Agent, we normalized addresses on the agent side. So no need to consider the mapping to normalize in the server-side for the profiles generated by the agent.

```
// NOTICE: We only store a single version of a mapping.
// Which means the m.Start actually correct for a single process.
// For a multi-process shared library, this will always be wrong.
// And storing the mapping for each process will be very expensive.
```

### CODE REVIEW

File: pkg/profile/profile_flat.go [hunk 1]
```diff
@@ -250,7 +258,11 @@ func (pn *profileFlatNormalizer) mapMapping(ctx context.Context, src *profile.Ma
 		return mapInfo{}, err
 	}
 	if m != nil {
-		mi := mapInfo{m, int64(m.Start) - int64(src.Start)}
+		// NOTICE: We only store a single version of a mapping.
```
Comment (kakkoyun) [inline pkg/profile/profile_flat.go:261]: @brancz I think this is the most important bit.

File: pkg/metastore/metastore_test.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func TestMappingKeyBytes(t *testing.T) {
 		Offset: 2,
 	}
 
-	require.Equal(t, MakeMappingKey(m), []byte{
+	require.Equal(t, []byte{
```
Comment (kakkoyun) [inline pkg/metastore/metastore_test.go:34]: This has changed because expected should be first and actual the second.

**Comment by kakkoyun** [comment]
I think actions are completely out-of-order right now.

**Review by brancz — APPROVED** [review]
I can't wait to see this work. 💯 🎉

**Pull request merged**